components/elm/src/biogeochem/MaintenanceRespMod.F90

```diff
@@ -159,8 +195,13 @@ subroutine MaintenanceResp(bounds, &
 
             ! calculate temperature corrections for each soil layer, for use in
             ! estimating fine root maintenance respiration with depth
-            tcsoi(c,j) = Q10**((t_soisno(c,j)-SHR_CONST_TKFRZ - 20.0_r8)/10.0_r8)
-        
+            ! Ben Sulman: Adding lower dormant maintenance resp below a certain
+            ! temperature
+            if (t_soisno(c,j) > dormant_mr_temp) then
```

Comment:
Be careful about what factor could actually increase respiration (depending on what Q10 function is equal to at the dormancy temperature). Could specify minimum of the two functions